elements/README.md

```diff
@@ -6,7 +6,7 @@
 
 ## What is Lisk Elements
 
-Lisk Elements is a collection of libraries, each of them implementing some form of blockchain application functionality such as cryptography, transactions, p2p, etc. Each library is designed to be compatible with the [Lisk Protocol](https://lisk.io/documentation/lisk-protocol).
+Lisk Elements is a collection of libraries, each of them implementing some form of blockchain application functionality such as cryptography, transactions, p2p, etc. Each library is designed to be compatible with the [Lisk Protocol](https://research.lisk.com/).
```

Comment:
Any reason for changing this to `research.lisk.com` ?
I don't see anything that goes by `Lisk Protocol` here.

How about `https://lisk.com/documentation/lisk-sdk/protocol/index.html`